ports/geckolib/glue.rs

```diff
@@ -290,15 +292,15 @@ pub extern "C" fn Servo_GetComputedValuesForAnonymousBox(parent_style_or_null: *
         Ok(pseudo) => pseudo,
         Err(pseudo) => {
             warn!("stylo: Unable to parse anonymous-box pseudo-element: {}", pseudo);
-            return ptr::null_mut();
+            return ComputedValues::null_strong();
         }
     };
 
     type Helpers = ArcHelpers<ServoComputedValues, ComputedValues>;
 
     Helpers::maybe_with(parent_style_or_null, |maybe_parent| {
         let new_computed = data.stylist.precomputed_values_for_pseudo(&pseudo, maybe_parent);
-        new_computed.map_or(ptr::null_mut(), |c| Helpers::from(c))
+        new_computed.map_or(ComputedValues::null_strong(), |c| HasStrong::into_strong(c))
```

Comment:
Would it make sense to have a method on HasStrong (this one could be default I believe) which does this for you? It seems like it's likely to be a common pattern (Option<Arc<Thing>> => Owning null or owning pointer)